content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,521 @@
+---
+title: "Reference Application using RDT for COVID-19 Testing"
+linkTitle: "COVID-19 Testing"
+weight: 
+description: >
+ A customizable CHT application with support for 3rd party RDT integration   
+relatedContent: >
+  apps/reference/forms/app/#android-app-launcher
+  apps/reference/forms/app/#cht-xform-widgets
+  apps/reference/forms/app/#parse-timestamp-to-date
+---
+
+Medic has worked with [FIND](https://www.finddx.org) to provide a reference application using rapid diagnostic tests (RDT) for COVID-19 testing with the CHT.  By leveraging the expertise of third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), you can customize this application in your current or future deployment of the CHT. 
```

Comment:
When rereading this it seems as though it could use a bit more context to understand _why_ you'd want to use third-party applications.

```suggestion
Medic has worked with [FIND](https://www.finddx.org) to build a CHT reference application for COVID-19 point-of-care testing with Rapid Diagnostic Tests (RDT). Using the reference app as an example, CHT app developers can easily include the provisioning and capture of RDT in workflows. These workflows can include third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), that guide health workers through the use of the RDT.
```